src/Symfony/Component/Console/Application.php

```diff
@@ -901,6 +901,32 @@ protected function doRenderThrowable(\Throwable $e, OutputInterface $output): vo
         } while ($e = $e->getPrevious());
     }
 
+    /**
+     * Get shell verbosity.
+     *
+     * @param InputInterface $input
```

Comment:
This `@param` and `@return` doc has no benefit. IMHO the whole phpdoc can be removed.

Reply:
Removed phpdoc